packages/cyberchef.vm/tools/chocolateyinstall.ps1

```diff
@@ -19,13 +19,23 @@ try {
   Install-ChocolateyZipPackage @packageArgs
   VM-Assert-Path $toolDir
 
+  # Create .ico for cyberchef icon
+  Add-Type -AssemblyName System.Drawing
```

Comment:
[nitpick] Could we need this feature (converting a .png to .ico) for other packages? Maybe worth to move this code to a helper in common. :thinking: